.github/workflows/go.yml

```diff
@@ -22,15 +22,23 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
-    - name: Get dependencies
+    - name: Diff
       run: |
-        go get -v -t -d ./...
-        if [ -f Gopkg.toml ]; then
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-        fi
+        go get golang.org/x/tools/cmd/goimports
```

Comment:
That is already done by `make init`. So we only have to move "Get dependencies" (not only linters) up.